chassembly/src/main/java/org/quiltmc/chasm/lang/api/ast/ReferenceNode.java

```diff
@@ -50,12 +51,19 @@ public Node evaluate(Evaluator evaluator) {
     }
 
     @Override
-    public void render(Renderer renderer, StringBuilder builder, int currentIndentationMultiplier) {
+    public void render(Renderer renderer, StringBuilder builder, int indentation, OperatorPriority minPriority) {
+        boolean needsBrackets = !OperatorPriority.ARGUMENT_PRIMARY.allowedFor(minPriority);
+        if (needsBrackets) {
```

Comment:
Why would a reference ever need to be enclosed in brackets?

Reply:
Yeah, I added these for completeness sake, I can remove them.

Reply:
I'd rather not have dead code.
If you want to have completeness, feel free to replace it with a comment mentioning that the check is unnecessary.